### ISSUE TITLE
Fix Luau types getting collapsed to one line

### ISSUE THREAD

**Issue by vocksel** [JohnnyMorganz/StyLua#277]
When run on a script containing Luau type, StyLua turns this:

```lua
export type Receptionist = Model & {
	ProximityPrompt: ProximityPrompt,
}
```
Into this:
```lua
export type Receptionist = Model & { ProximityPrompt: ProximityPrompt }
```

This is making it hard to read types, especially when they contain several members

**Comment by JohnnyMorganz** [comment]
What version of StyLua are you running? This should've been fixed in 0.11.0 (#226)

**Comment by vocksel** [comment]
Was running 0.10.0, upgraded and this issue is resolved :)
